src/DependencyInjection/Compiler/EventAliasMappingPass.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the PrestaSitemapBundle package.
+ *
+ * (c) PrestaConcept <https://prestaconcept.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Presta\SitemapBundle\DependencyInjection\Compiler;
+
+use Presta\SitemapBundle\Event\SitemapAddUrlEvent;
+use Presta\SitemapBundle\Event\SitemapPopulateEvent;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Maps event classes to event names
+ */
+final class EventAliasMappingPass implements CompilerPassInterface
```

Comment:
After a quick search over Symfony's source, I've found that they already wrote a compiler pass for this : [`AddEventAliasesPass`](https://github.com/symfony/event-dispatcher/blob/5.4/DependencyInjection/AddEventAliasesPass.php)

I think we should use it instead of writing our own

Reply:
So revert to previous way?